docs/user-guide/lns-discovery.md

```diff
@@ -0,0 +1,65 @@
+# LNS discovery
+
+The LNS protocol specifies that the first request an LBS makes is to the `/router-info` discovery endpoint. The LNS from the starter kit implements the `/router-info` endpoint and redirects any requests to that endpoint to the `/router-data` endpoint on the same LNS.
+
+In addition to the discovery endpoint on the LNS itself, we also support a standalone discovery service, which you can use to dynamically distribute LBS connections to different LNS, ensuring failover of LBS to another LNS in case the current LNS is experiencing downtime. The discovery service is a .NET application, and its implementation is part of the `LoRaWan.NetworkServerDiscovery` project.
+
+The discovery service relies on configuration from IoT Hub module twins to associate each LBS with a set of LNS to which it may connect to. For this the configuration needs to take into account possible network boundaries - by defining these networks you can control which LBS should connect to which LNS. For this you need to configure your LBS twin in IoT Hub as follows:
+
+```json
+{
+  ...,
+  "tags": {
+    "network": "<network-id>"
+  },
+  ...
+}
+```
+
+The network ID may only consist of alphanumeric characters. After you configured the LBS, you need to also configure your LNS with the same network. The LNS twin furthermore needs to contain the IP/DNS of the host, hence the twin of the LNS *module* should look like:
+
+```json
+{
+  ...,
+  "tags": {
+    "network": "<network-id>"
+  },
+  ...,
+  "properties": {
+    "desired": {
+      "hostAddress": "wss://mylns:5001",
+      ...
+    }
+  }
+}
+```
+
+Keep in mind that the `hostAddress` value must include the scheme and the host name. In case you omit the port, the defaults of the protocol will be used.
```

Comment:
What about asking to always specify the port too?
Default of the protocol are useless in our starter kit

Reply:
It would be stricter, but it would also require code changes. I would argue that if they change the port mappings that we use per default in our starter kit, it might still be useful to keep it.